README.md

```diff
@@ -351,7 +351,7 @@ you can then use the “uiSpin.state” property inside the view to display data
 Button(action: {
     self.uiSpin.emit(Event.startCounter)
 }) {
-    Text("\(self.uiSpin.state.isCounterPaused ? "Start": "Stop")")
+    Text("\(self.uiSpin.state.isCounterPaused? "Start": "Stop")")
```

Comment:
The ? Cannot be right after the “isCounterPaused” because the compiler will interpret it as an optional instead of a ternary operator

Reply:
True.  In my mind, I hadn't switch from reading text to code.